### PR TITLE
Lk dragen methods doc

### PR DESCRIPTION
**Pull request by ekiernan**
Added a new section to the methods doc for DRAGEN Functional Equilvaence mode

### CODE REVIEW

File: website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md [hunk 1]
```diff
@@ -32,6 +32,30 @@ Prior to variant calling, the variant calling interval list was split to enable
 
 The pipeline’s final outputs included metrics, validation reports, an aligned CRAM with index, and a reblocked GVCF containing variant calls with an accompanying index.
 
+## Detailed methods for the Functional Equivalence mode of the Whole Genome Germline Single Sample workflow
+
+Preprocessing and variant calling was performed using the WholeGenomeGermlineSingleSample 3.0.0 pipeline using Picard 2.23.8, GATK 4.2.2.0, and Samtools 1.11 with default tool parameters unless otherwise specified. All reference files are available in the public [Broad References Google Bucket](https://console.cloud.google.com/storage/browser/gcp-public-data--broad-references/hg38/v0). The pipeline follows the Functional Equivalence specification ([Regier et al., 2018](https://www.nature.com/articles/s41467-018-06159-4)).
```
Comment (michaelgatzen) [inline website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md:37]: Can we link to this article instead?
https://gatk.broadinstitute.org/hc/en-us/articles/4410456501915-Functional-equivalence-in-DRAGEN-GATK

File: website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md [hunk 2]
```diff
@@ -32,6 +32,30 @@ Prior to variant calling, the variant calling interval list was split to enable
...
+
+### Pre-processing and quality control metrics
+
+Whole genome paired-end reads in unmapped BAM (uBAM) format were first scattered to perform QC and alignment in parallel. Quality metrics were calculated using Picard CollectQualityYieldMetrics. uBAMs were converted to FASTQ using Picard SamToFastq and aligned to the Hg38 reference genome (reference files available at gs://gcp-public-data--broad-references/hg38/v0/dragen_reference/) using the DRAGMAP aligner with --interleaved=1. Metadata from the uBAMs was then merged with the aligned BAMs using Picard MergeBamAlignment with the parameters --SORT_ORDER="unsorted", allowing the data to be grouped by read name for efficient downstream marking of duplicates, and --UNMAP_CONTAMINANT_READS=false, to maintain functional equivalence to the DRAGEN hardware.
```
Comment (michaelgatzen) [inline website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md:41]:
```suggestion
Whole genome paired-end reads in unmapped BAM (uBAM) format were first scattered to perform QC and alignment in parallel. Quality metrics were calculated using Picard CollectQualityYieldMetrics. uBAMs were converted to FASTQ using Picard SamToFastq and aligned to a masked version of the hg38 reference genome (reference files available at gs://gcp-public-data--broad-references/hg38/v0/dragen_reference/) using the DRAGMAP aligner. Metadata from the uBAMs was then merged with the aligned BAMs using Picard MergeBamAlignment with the parameter --UNMAP_CONTAMINANT_READS=false to maintain functional equivalence to the DRAGEN hardware.
```

Comment (michaelgatzen) [reply website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md:41]: The `interleaved` and `SORT_ORDER` arguments don't have any interesting meaning, they are just needed to make input/output formats of the different tools compatible with each other, so I think we can leave this out

File: website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md [hunk 3]
```diff
@@ -32,6 +32,30 @@ Prior to variant calling, the variant calling interval list was split to enable
...
+
+To evaluate the coverage and performance of the whole genome sequencing experiment, the BAM was assessed using the Picard tools CollectWGSMetrics and CollectRawWgsMetrics.
+
+The BAM was converted to CRAM using Samtools view and validated using Picard ValidateSamFile.
+
+### Variant calling
+
+Prior to variant calling, the DRAGEN STR model was calibrated using the CalibrateDragstrModel tool with a [public STR table](gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.str). The variant calling interval list was then split to enable parallelization. Variant calling was applied to reads using GATK4 HaplotypeCaller with the parameters --dragen-mode --disable-spanning-event-genotyping, and --ERC GVCF. The resulting GVCFs were merged using Picard MergeVcfs and then reblocked using GATK ReblockGVCF with -GQB 20 -GQB 30 -GQB 40. The final reblocked GVCF file was validated using GATK ValidateVariants. Variant metrics were calculated using Picard CollectVariantCallingMetrics.
```
Comment (michaelgatzen) [inline website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md:55]:
```suggestion
Prior to variant calling, the DRAGEN STR model was calibrated using the CalibrateDragstrModel tool with a [public STR table](gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.str). The variant calling interval list was then split to enable parallelization. Variant calling was applied to reads using GATK4 HaplotypeCaller with the parameters --dragen-mode --disable-spanning-event-genotyping (to maintain functional equivalence to the DRAGEN hardware), and --ERC GVCF. The resulting GVCFs were merged using Picard MergeVcfs and then reblocked using GATK ReblockGVCF with -GQB 20 -GQB 30 -GQB 40. The final reblocked GVCF file was validated using GATK ValidateVariants. Variant metrics were calculated using Picard CollectVariantCallingMetrics.
```

File: website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md (hunk 1 above)
Comment (ekiernan) [reply website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md]: Yes, I added it. Thank you!

**Review by michaelgatzen — APPROVED** [review]

**Review by jessicaway — APPROVED** [review]
Looks good!

**Review by wjdingman1 — APPROVED** [review]

**Pull request merged**